common/rya.api/src/main/java/mvm/rya/api/persist/index/RyaSecondaryIndexer.java

```diff
@@ -34,7 +34,14 @@
 
 public interface RyaSecondaryIndexer extends Closeable, Flushable, Configurable {
 
-    public String getTableName();
+	/**
+	 * 
+	 * Returns the table name if the implementation supports it.
+	 * Note that some indexers use multiple tables, this only returns one.
```

Comment:
Maybe put a TODO here suggesting we deprecate this method because it's a leaky interface.